.devcontainer/devcontainer.json

```diff
@@ -1,52 +1,50 @@
-		"python.linting.pycodestylePath": "/usr/local/py-utils/bin/pycodestyle",
-		"python.linting.pydocstylePath": "/usr/local/py-utils/bin/pydocstyle",
-		"python.linting.pylintPath": "/usr/local/py-utils/bin/pylint",
-		"lldb.executable": "/usr/bin/lldb",
-		"files.watcherExclude": {
-			"**/target/**": true
-		}
-	},
-	"remoteUser": "codespace",
-	"overrideCommand": false,
-	"workspaceMount": "source=${localWorkspaceFolder},target=/home/codespace/workspace,type=bind,consistency=cached",
-	"workspaceFolder": "/home/codespace/workspace",
-	"mounts": [ "source=/var/run/docker.sock,target=/var/run/docker-host.sock,type=bind" ],
-	"runArgs": [ "--cap-add=SYS_PTRACE", "--security-opt", "seccomp=unconfined" ],
+  "name": "GitHub Codespaces (Default)",
+  "build": {
+    "dockerfile": "Dockerfile",
+    "args": {
+      "INSTALL_DOCKER": "true"
+    }
+  },
+  "settings": {
+    "terminal.integrated.shell.linux": "/bin/bash",
+    "go.useGoProxyToCheckForToolUpdates": false,
+    "go.useLanguageServer": true,
+    "go.gopath": "/go",
+    "go.goroot": "/usr/local/go",
+    "go.toolsGopath": "/go/bin",
+    "python.pythonPath": "/opt/python/latest/bin/python",
+    "python.linting.enabled": true,
+    "python.linting.pylintEnabled": true,
+    "python.formatting.autopep8Path": "/usr/local/py-utils/bin/autopep8",
+    "python.formatting.blackPath": "/usr/local/py-utils/bin/black",
+    "python.formatting.yapfPath": "/usr/local/py-utils/bin/yapf",
+    "python.linting.banditPath": "/usr/local/py-utils/bin/bandit",
+    "python.linting.flake8Path": "/usr/local/py-utils/bin/flake8",
+    "python.linting.mypyPath": "/usr/local/py-utils/bin/mypy",
+    "python.linting.pycodestylePath": "/usr/local/py-utils/bin/pycodestyle",
+    "python.linting.pydocstylePath": "/usr/local/py-utils/bin/pydocstyle",
+    "python.linting.pylintPath": "/usr/local/py-utils/bin/pylint",
+    "lldb.executable": "/usr/bin/lldb",
+    "files.watcherExclude": {
+      "**/target/**": true
+    }
+  },
+  "remoteUser": "codespace",
+  "overrideCommand": false,
+  "workspaceMount": "source=${localWorkspaceFolder},target=/home/codespace/workspace,type=bind,consistency=cached",
+  "workspaceFolder": "/home/codespace/workspace",
+  "mounts": ["source=/var/run/docker.sock,target=/var/run/docker-host.sock,type=bind"],
+  "runArgs": ["--cap-add=SYS_PTRACE", "--security-opt", "seccomp=unconfined"],
 
-	// Add the IDs of extensions you want installed when the container is created.
-	"extensions": [
-		"GitHub.vscode-pull-request-github",
-		"MS-vsliveshare.vsliveshare",
-		"VisualStudioExptTeam.vscodeintellicode"
-	]
+  // Add the IDs of extensions you want installed when the container is created.
+  "extensions": [
+    "GitHub.vscode-pull-request-github",
+    "MS-vsliveshare.vsliveshare",
+    "VisualStudioExptTeam.vscodeintellicode"
+  ],
+  "postCreateCommand": "" // Disable Oryx build which runs npm install and creates a phantom node_modules folder
```

Comment:
perhaps do `rush update` here?

Reply:
I considered that - and maybe we should. My concern was I didn't want any build time errors from preventing codespace creation. If you have a bad dependency you'd still want to be able to get in and edit your code. 

I also noticed that Oryx still built the project, but according to these docs it shouldn't have so I have to hunt down why that didn't work https://docs.microsoft.com/en-us/visualstudio/codespaces/reference/configuring#general-properties (I realize that's VS Codespaces but the same command is available in VSCode docs)

Reply:
> My concern was I didn't want any build time errors from preventing codespace creation. If you have a bad dependency you'd still want to be able to get in and edit your code.

I see. I think you can use codespaces with branches other than master where build-time errors can be tolerated so not doing `rush update` should be the right thing to do.